docs/reading_files.rst

```diff
@@ -268,3 +268,34 @@ You only need these details if you think caching may be causing problems.
 
 JSON was chosen as it can be easily inspected manually, and it's reasonably
 efficient to load the entire file.
+
+File Reading Issues
+-------------------
+
+Files at European XFEL storage migrate over time from GPFS to PNFS. The data 
+on PNFS usually always available for reading. But sometimes, reading may require
+staging from the tape to dCache. If there is staging queue, the operation can take
+an indefinitely long time (days or even weeks) and any IO operations will be 
+blocked for this time.
+
+To determine the files which require staging or are lost, use the script::
+
+    extra-date-locality <run directory>
+
+It returns a list of files which are currently located only on slow media for some
+reasons and, separetelly, which are lost at all.
+
+If the files are not essential for analysis, then they can be filtered out using 
+filter :func:`ls_ondisk` from :mod:`extra_data.locality`::
+
+    from extra_data.locality import lc_ondisk
+    run = open_run(proposal=700000, run=1, file_filter=lc_ondisk)
+
+``file_filter`` must be a callable which takes a list as a single argument and
+returns filtered list.
+
+**Note: the reading of file locality on PNFS very expensive operation.
+Use it only as a last resort.**
+
+If problem files are essential for analysis, contact `IT service <mailto:it-support@xfel.eu>`_
```

Comment:
```suggestion
If unavailable files are essential for analysis, contact `IT service <mailto:it-support@xfel.eu>`_
```

I think we should make it clear that data on tape is not a problem from the facility's point of view - it's meant to happen, and you can still access it, you just need to be patient.

Reply:
I learned yesterday that for the time being, ITDM is keeping all raw data in the disk cache, and we can let them know if that's not the case. I don't know how long that holds for, though, and it doesn't cover all the calibrated data.

Reply:
That the reason why I would like to encourage users to give feedback to ITDM every time. It may give them a clue how dCache issues affect on users. What do you think about following:

```suggestion
If you find any files which are located only on tape or unavailable, please let know to
`ITDM <mailto:it-support@xfel.eu>`_. If you need these files for analysis mentioned
that explicitly.
```
I agree with other suggestions. Thanks.

Reply:
That sounds OK. We can always adjust it later if IT complain that too many people are contacting them.